numpy/core/tests/test_multiarray.py

```diff
@@ -1291,26 +1298,29 @@ def test_structured_comparisons_with_promotion(self):
         assert_equal(a == b, [False, True])
         assert_equal(a != b, [True, False])
 
-    def test_void_comparison_failures(self):
+    @pytest.mark.parametrize("op", [
+            operator.eq, lambda x, y: operator.eq(y, x),
```

Comment:
Not quite sure why the `lambda`s are also tested...

Reply:
They test the reverse direction.

Reply:
Ah, missed that...